docs/source/errors.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Errors
+subtitle: Error and status codes returned by the Apollo Router
+description: Error codes and response status codes returned by the Apollo Router
+---
+
+Learn about error codes and HTTP response status codes returned by the Apollo Router.
+
+## Status codes
+
+<PropertyList kind="errCodes">
+<Property name="400" short="Bad request">
+
+A request failed GraphQL validation, or a request failed to be parsed.
```

Comment:
Definite articles seem more natural to me? Will just make the suggestion here rather than globally.

```suggestion
The request failed GraphQL validation or failed to be parsed.
```